### PR TITLE
framenet.py: doctest fixes for Python 3.x compatibility

### PR DESCRIPTION
**Pull request by nschneid**
see #479

### CODE REVIEW

File: nltk/corpus/reader/framenet.py [hunk 1]
```diff
@@ -1328,16 +1328,16 @@ def documents(self, name=None):
...
-        set(['ANC',
-             'C-4',
-             'KBEval',
-             'LUCorpus-v0.3',
-             'Miscellaneous',
-             'NTI',
-             'PropBank',
-             'QA',
-             'SemAnno'])
+        >>> pprint({x.corpname for x in fn.documents()})
```
Comment (kmike) [inline nltk/corpus/reader/framenet.py]: this syntax is not supported in Python 2.6

File: nltk/corpus/reader/framenet.py [hunk 2]
```diff
@@ -1329,15 +1329,15 @@ def documents(self, name=None):
...
-        set(['ANC',
-             'C-4',
-             'KBEval',
-             'LUCorpus-v0.3',
-             'Miscellaneous',
-             'NTI',
-             'PropBank',
-             'QA',
-             'SemAnno'])
+        {'ANC',
```
Comment (kmike) [inline nltk/corpus/reader/framenet.py]: I bet this breaks tests in Python 2.x :) Unfortunately set repr is different in 2.x and 3.x, and NLTK test runner doesn't fix that. It seems that comparing two sets is the sanest way to deal with this issue.

File: nltk/corpus/reader/framenet.py [hunk 3]
```diff
@@ -1513,8 +1513,8 @@ def semtypes(self):
         >>> stypes = fn.semtypes()
         >>> len(stypes)
         73
-        >>> stypes[0].keys()
-        ['definition', '_type', 'name', 'rootType', 'abbrev', 'superType', 'subTypes', 'ID']
+        >>> list(sorted(stypes[0].keys()))
```
Comment (kmike) [inline nltk/corpus/reader/framenet.py]: list(..) is unnecessary because `sorted` returns list

**Comment by nschneid** [comment]
Oops, I wasn't testing the file I thought I was testing. Tests now pass on 2.7.

**Comment by kmike** [comment]
Thanks! 
One test is still failing in Python 3.3 for me; I'll update #479.

**Pull request merged**